<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="todoListPage">
     <widget class="QTableView" name="ToDoList">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>471</width>
        <height>451</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(109, 109, 109);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="toDoListBack">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>500</width>
        <height>470</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 27%;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>toDoListBack</zorder>
     <zorder>ToDoList</zorder>
    </widget>
    <widget class="QWidget" name="mapPage">
     <widget class="QLabel" name="MapBackground">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>500</width>
        <height>470</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 27%</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="MapLoadButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>424</x>
        <y>89</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#MapLoadButton{
background-color: rgb(100, 100, 100);
border-radius: 25%;
color: rgb(235, 235, 235);
}
QPushButton#MapLoadButton:hover
{
	
	border-color: rgb(30, 39, 67);
	
	
	background-color: rgb(130, 156, 208);
	color:rgb(204, 204, 204);
}</string>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="AreaCodeLine">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>89</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(100, 100, 100);
border-color: rgb(109, 109, 109);
color: rgb(235, 235, 235);
border-radius:99%;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Map">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>461</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 27%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="imageDownloadPage">
     <widget class="QLabel" name="imageDownload">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>500</width>
        <height>470</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(109, 109, 109);
border-radius: 27%;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="localHourLCD">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>118</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(50, 50, 50)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="localMinuteLCD">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>118</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(50, 50, 50)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="localSecondLCD">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>118</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(50, 50, 50)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="Weather">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(109, 109, 109);
border-radius:27%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="colonOne">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>118</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(109, 109, 109);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colonTwo">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>118</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(109, 109, 109);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colonOne_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>10</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(109, 109, 109);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="colonTwo_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>10</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(109, 109, 109);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldHourLCD">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>30</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(50, 50, 50)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldMinuteLCD">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>30</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(50, 50, 50)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldSecondLCD">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>30</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);
background-color: rgb(50, 50, 50);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
   </widget>
   <widget class="QRadioButton" name="page1">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>540</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="page2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="page3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="forcastWeatherPicture">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(109, 109, 109);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="GreetingLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>310</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(36, 195, 197);
background-color: rgb(50, 50, 50)</string>
    </property>
    <property name="text">
     <string>Welcome!</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
border-color: rgb(109, 109, 109);
border-radius:99%;
color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherDownloadButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#weatherDownloadButton{
background-color: rgb(100, 100, 100);
border-radius: 25%;
color: rgb(235, 235, 235);
}
QPushButton#weatherDownloadButton:hover
{
	
	border-color: rgb(30, 39, 67);
	
	
	background-color: rgb(130, 156, 208);
	color:rgb(0, 0, 0)
}</string>
    </property>
    <property name="text">
     <string>See Forecast</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>190</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(235, 235, 235);</string>
    </property>
    <property name="text">
     <string>Switzerland</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
